content-services/latest/config/mtls.md

```diff
@@ -208,12 +208,21 @@ The default settings for `transform` are shown below:
 httpclient.config.transform.mTLSEnabled=false
 httpclient.config.transform.maxTotalConnections=20
 httpclient.config.transform.maxHostConnections=20
-httpclient.config.transform.socketTimeout=5000

```

Comment:
In that case I'd change "The default settings for `transform` are shown below:" to "The default settings for `transform` are shown below, timeouts are infinite by default:"

Reply:
I've seen more things, I'll provide a fix myself along with a few other changes